### PR TITLE
added two colab files under the quickstart folder in examples

### PR DESCRIPTION
**Pull request by carolinef35**
## Description

Added two colab notebooks based on "influence_function_noisy_label.py" and "influence_function_lds.py". The notebooks have descriptive comments and are set up to run in under 5 minutes making their use efficient for new users of the dattri library.

### CODE REVIEW

File: examples/quickstart/influence_function_lds.ipynb [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    "language_info": {
+      "name": "python"
+    },
+    "accelerator": "GPU"
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# This example shows how to use the IF to calculate LDS scores to detect noisy labels in the MNIST dataset."
```
Comment (jiaqima) [inline examples/quickstart/influence_function_lds.ipynb]: This notebook is irrelevant to noisy label detection.

File: examples/quickstart/influence_function_lds.ipynb [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+        "from dattri.task import AttributionTask"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "LDS Score: metric used to evaluate the influence or importance of individual training data points on the model's predictions for specific test data points.\n",
+        "\n",
+        "\n",
+        "*   A higher LDS score for a training data point suggests that removing or perturbing that data point would have a more significant impact on the model's behavior or predictions.\n",
```
Comment (jiaqima) [inline examples/quickstart/influence_function_lds.ipynb]: This is inaccurate

File: examples/quickstart/influence_function_lds.ipynb [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+        "                sampler=model_details[\"train_sampler\"],\n",
+        "            ),\n",
+        "            DataLoader(\n",
+        "                model_details[\"test_dataset\"],\n",
+        "                batch_size=5000,\n",
+        "                sampler=model_details[\"test_sampler\"],\n",
+        "            ),\n",
+        "        )\n",
+        "\n",
+        "    # calculate the LDS (Leave-one-out Data Sensitivity) score\n",
```
Comment (jiaqima) [inline examples/quickstart/influence_function_lds.ipynb]: LDS != Leave-one-out Data Sensitivity

**Comment by jiaqima** [comment]
Please open a new PR since substantial correction is needed.
